File: 520 Detect Capital.java
class Solution {
    public boolean detectCapitalUse(String word) {

        if (word.length() < 2) return true;

        if (word.toUpperCase().equals(word)) return true;

        // substring()只有一个参数时表示开始index
        if (word.substring(1).toLowerCase().equals(word.substring(1))) return true;

        return false;
    }
}

File: 520 Detect Capital.py
class Solution:
    def detectCapitalUse(self, word: str) -> bool:
        return word.isupper() or word.islower() or word.istitle()


'''
The isupper() methods returns “True” if all characters in the string are uppercase, Otherwise, It returns “False”
The islower() methods returns “True” if all characters in the string are lowercase, Otherwise, It returns “False”.
The istitle() methods returns “True” if all words in a text start with a upper case letter, AND the rest of the word are lower case letters, otherwise False.
'''
